exercises/two-bucket/canonical-data.json

```diff
@@ -96,6 +96,36 @@
         "goalBucket": "two",
         "otherBucket": 2
       }
+    },
+    {
+      "uuid": "449be72d-b10a-4f4b-a959-ca741e333b72",
+      "description": "Not possible to reach the goal",
+      "property": "measure",
+      "input": {
+        "bucketOne": 6,
+        "bucketTwo": 15,
+        "goal": 5,
+        "startBucket": "one"
+      },
+      "expected": {
+        "error": "impossible"
+      }
+    },
```

Comment:
How about a test where the goal is bigger than the biggest bucket?
```json
    {
      "uuid": "...",
      "description": "Goal cannot be reached",
      "property": "measure",
      "input": {
        "bucketOne": 5,
        "bucketTwo": 7,
        "goal": 8,
        "startBucket": "one"
      },
      "expected": {
        "error": "impossible"
      }
    },
```

Reply:
I like it. Added the case as-is.

Reply:
Thanks!